gitlabform/gitlab/core.py

```diff
@@ -98,10 +98,6 @@ def _get_project_id(self, project_and_group):
         result = self.get_project(project_and_group)
         return str(result["id"])
 
-    def has_no_license(self):
-        license = self._make_requests_to_api("license")
-        return not license or license["expired"]
```

Comment:
Moved to a test fixture so markers can be used without instantiating a separate helper client.